### DISCUSSION TITLE
Computer performance problems

### DISCUSSION THREAD

**Discussion by danghai255** [engageub/InternetIncome#99] [Q&A]
Hello everyone!
-Please let me know how many proxies can run with Ubuntu 8c16G VPS?
-My personal computer at home has a Dual E5-2680v4 28cores 56 processors , 128G ram configuration running VMware Ubuntu and can only run a maximum of 50 proxies at full load.
-I run using Datacenter Proxies

**Comment by engageub** [comment]
Hello,
It depends on the frequency of the processor also and type of containers you are running. 
You have to calculate accordingly based on the existing cpu. 
If you are using firefox containers like ebesucher and adnade, they consume more cpu and ram than others

Thank you

**Reply by danghai255** [reply]
Hello
Thank you very much for your reply. I will try testing without using those two applications to see if the device's performance improves.

**Reply by engageub** [reply]
To check the consumption for each container, you may run the following command

```
sudo docker stats
```

For more options please click the link [here](https://docs.docker.com/engine/reference/commandline/container_stats/#:~:text=The%20docker%20stats%20command%20returns,do%20not%20return%20any%20data.).

**Reply by danghai255** [reply]
It seems my problem has been resolved. Those 2 apps really used too much resources, I deleted them.
Thank you very much!
